### ISSUE TITLE
FIPS_UNITTESTS ON also activates unittests for oryol

### ISSUE THREAD

**Issue by mgerhardy** [floooh/fips#121]
Usually I don't wanna run the unittests of my dependencies - because I don't modify them directly. A option to disable dependency related unittests would be awesome.

**Comment by mgerhardy** [comment]
oh... and another thing. If I activate FIPS_UNITTESTS in my CMakeLists.txt, I get unresolved symbols for e.g.

```
[ 27%] Linking CXX executable /home/mgerhardy/dev/fips-deploy/wov/linux-make-debug/TimeTest
/usr/bin/ld: cannot find -lunittestpp
```

mentioning the unittestpp dep in my fips.yml doesn't solve the problem, too

**Comment by floooh** [comment]
Yeah, there are many unsolved problems with FIPS_UNITTEST, it is basically hardwired to unittestpp (for instance: https://github.com/floooh/fips/blob/master/cmake/fips_unittests.cmake#L44), it probably makes more sense to remove the whole unittest stuff from fips since it is very project specific (e.g. what testing framework is used etc...).

You're probably better off adding your own cmake option and wrapper macros.

**Comment by fungos** [comment]
I agree. But at the same time I think that a FIPS_TESTS and a target tests should exists in fips. So we can hook to it with unittest or gtest for example.
